client/homebrew/pages/editPage/editPage.jsx

```diff
@@ -205,8 +200,9 @@ const EditPage = createClass({
 			.send(brew)
 			.catch((err)=>{
 				console.log('Error Updating Local Brew');
-				this.setState({ errors: err });
+				this.setState({ error: err.response });
```

Comment:
Not sure if this line is the cause, but the error messages generated to send to GitHub is this on the live site:

![image](https://user-images.githubusercontent.com/5878534/214127772-569383d4-480c-42c4-bc96-8971153ac034.png)

But seems to be missing a lot of info with this PR:

![image](https://user-images.githubusercontent.com/5878534/214127900-1860c2b3-fa54-4b68-ac93-4e4fc5a0d531.png)

Reply:
<img width="903" alt="image" src="https://user-images.githubusercontent.com/4909307/214141785-621fb927-e47a-4e31-b17e-7099afdda4fa.png">
After my adjustments we get the stack trace, while it's not pretty I feel like it's more complete than it was before